examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -44,6 +44,39 @@
     "BATCH_SIZE = 64"
```

Comment:
<p>Add comments to this chunk</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>